jplag/src/main/java/de/jplag/reporting/reportobject/model/JPlagReport.java

```diff
@@ -1,10 +1,7 @@
 package de.jplag.reporting.reportobject.model;
 
 import java.util.List;
+import java.util.Map;
 
-public record JPlagReport(OverviewReport overviewReport, List<ComparisonReport> comparisons) {
-    public JPlagReport(OverviewReport overviewReport, List<ComparisonReport> comparisons) {
-        this.overviewReport = overviewReport;
-        this.comparisons = List.copyOf(comparisons);
-    }
+public record JPlagReport(OverviewReport overviewReport, List<ComparisonReport> comparisons, Map<Long, String> lineLookUpTable) {
```

Comment:
Was the change intended. Before it was a copy.

Reply:
This class has now been completely deleted